### PR TITLE
LOOM-1304 BpkNudger Class 2 Overrides

### PR DESCRIPTION
**Pull request by jronald01**
**Class 2 Overrides**:

1. Moved all `classname` overrides in `BpkNudger` into wrappers and re-jigged styling to compensate.

This ended-up being more involved as no matter what I did, I couldn't get the **zoomed in mobile view** to match the Percy snapshot.

So I moved the **title** and **subtitle** from within the `BpkLabel`, which is used for the person icon and moved them to the same level as the `BpkLabel` rather than being children.

This was required because when we move the **classname override to a wrapper**, we _lose_ the **flex** arrangement, as `BpkLabel` uses `display: block;` **_after_ the wrapper**, which breaks the arrangement:
```
@mixin bpk-label {
  display: block;
```

There are no doubt multiple ways to fix this, but the current version fixes for all visual tests, zoomed or not zoomed and mobile and desktop whereas the previous fixes I experimented with always had at least one snapshot not matching.

Due to the linting rules, I also had to remove the `classname` from the component and removed the now redundant test that checked for a `classname`. 

-- -- 

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: packages/bpk-component-nudger/src/BpkNudger.tsx [hunk 1]
```diff
@@ -59,24 +59,21 @@ const BpkNudger = ({
...
       )}
       <BpkConfigurableNudger
         inputClassName={getClassName('bpk-nudger__input--numeric')}
         compareValues={compareValues}
         incrementValue={incrementValue}
         decrementValue={decrementValue}
         formatValue={formatValue}
-        // TODO: className to be removed
-        // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-        className={className}
```
Comment (olliecurtis) [inline packages/bpk-component-nudger/src/BpkNudger.tsx:79]: If we are removing this here should we also remove it from the API and also this PR would then become a major as we are removing functionality that folks could be using?

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: packages/bpk-component-nudger/src/BpkNudger.tsx (hunk 1 above)
Comment (mungodewar) [reply packages/bpk-component-nudger/src/BpkNudger.tsx:79]: Agreed. I don't think we can remove these "consumer pass-through" `className` cases. Tbh quite unsure on how to tackle these long term. As you say Ollie they can only be removed through a breaking change.

Comment (jronald01) [reply packages/bpk-component-nudger/src/BpkNudger.tsx:79]: Ah sorry, my bad, I got confused about this failure: https://github.com/Skyscanner/backpack/actions/runs/8649230361/job/23715083356#step:8:134
```
/home/runner/work/backpack/backpack/packages/bpk-component-nudger/src/BpkNudger.tsx
Error:   77:9  error  Prop className is forbidden. See ./decisions/className.md  @skyscanner/rules/forbid-component-props
```

Comment (jronald01) [reply packages/bpk-component-nudger/src/BpkNudger.tsx:79]: Do I just need to disable the linter for the line? @olliecurtis @mungodewar

Comment (jronald01) [reply packages/bpk-component-nudger/src/BpkNudger.tsx:79]: Or do I add it to `.eslintrc.js` in an `allowedFor` rules?

Comment (mungodewar) [reply packages/bpk-component-nudger/src/BpkNudger.tsx:79]: Let's leave it inline till we work out what is to be done with these cases.

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**